src/Chapter1.hs

```diff
@@ -631,8 +635,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Integral a => a -> a
+sumLast2 n = let
+    ld = lastDigit
```

Comment:
Any reasons to use the renamed function? 🤔

Reply:
The task was to introduce variables with either "let" or "where", so i decided to make expression shorter